app/src/ui/history/merge-call-to-action-with-conflicts.tsx

```diff
@@ -62,59 +62,77 @@ export class MergeCallToActionWithConflicts extends React.Component<
     behindCount: number
   ) {
     if (mergeStatus === null || mergeStatus.kind === MergeResultKind.Loading) {
-      return (
-        <div className="merge-message merge-message-loading">
-          Checking for ability to merge automatically...
-        </div>
+      return this.renderLoadingMergeMessage()
+    } else if (mergeStatus.kind === MergeResultKind.Clean) {
```

Comment:
No need for `else` statement here since you return in each block. Normal `if` statement is fine, or use a `switch` statement.

Reply:
ah good point :)